projects/Mallard/src/helpers/files.ts

```diff
@@ -394,12 +394,16 @@ export const readIssueSummary = async (): Promise<IssueSummary[]> =>
 
 export const fetchAndStoreIssueSummary = async (): Promise<IssueSummary[]> => {
     const apiUrl = await getSetting('apiUrl')
+    const edition = await getSetting('edition')
+
+    const fetchIssueSummaryUrl = apiUrl + `${edition}/` + 'issues'
```

Comment:
```suggestion
    const fetchIssueSummaryUrl = `${apiUrl}${edition}/issues`
```
Less of a mix of styles this way.